libtransmission/utils.cc

```diff
@@ -193,6 +196,15 @@ bool tr_file_save(std::string_view filename, std::string_view contents, tr_error
     {
         return false;
     }
+#ifndef _WIN32
+    // set file mode per settings umask()
+    else
+    {
+	mode_t val = umask(0);
```

Comment:
Do use `::umask` consistently.

You have both just `umask` and explicit `::umask` here.

Reply:
Corrected.  Explicit ::umask used.